.github/workflows/changelog.yml

```diff
@@ -0,0 +1,94 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+
+permissions:
+  contents: read
```

Comment:
Is there a reason we don't use the default `read|write` here?

Reply:
I took this over from Pierre's PR as well. We don't need write permissions here, so I guess this would just be the principle of least privileges. However I would be fine with removing this as well.

Reply:
👍 

Let's remove in the spirit of less is more then. I don't like how these simple workflow files can become quite long just to get their set up right.

Reply:
I wholeheartedly agree. It's already quite verbose just for the caching of the venv. 😞